packages/temporal/mpe/workflows/mpe.go

```diff
@@ -369,6 +369,8 @@ func MpeRoomWorkflow(ctx workflow.Context, params shared_mpe.MpeRoomParameters)
 						FromIndex:        message.FromIndex,
 					}),
 				)
+			default:
+				panic("Encountered an unkown MPE workflow signal")
```

Comment:
`panic` can also take an `error` as parameter.
It allows to do not use `strings.Contains` to know what was is the cause.